### PR TITLE
openssl-lts: Add version 1.1.1L (LTS)

### PR DESCRIPTION
**Pull request by GProulx**
## What type of Apps related PR is this? (check all applicable)

- [X] ✨ Add Manifest
- [ ] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

### Add/Modify a description in the manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Update Hashes?

- [X] 👍 yes
- [] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [X] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [X] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

:lock_with_ink_pen: ➕ :memo: :memo: :memo:

### CODE REVIEW

File: bucket/openssl-v1-lts.json
Comment (alexchapar) [inline bucket/openssl-v1-lts.json:27]: the checkversion could be something like "Win32 OpenSSL v(1(\\­.\\d)+[a-zA-Z])" which will limit to the autoupdate to only the "v1"

Comment (GProulx) [reply bucket/openssl-v1-lts.json]: Maybe not the prettiest regex (doesn't know if it's a possible thing! 🤔 🤯 ) but here what I found that match only v1.x and that the checkver.ps1 script is happy with! 🎉 

```
"checkver": "(1_\\d_\\d.)",
```

**Comment by alexchapar** [comment]
After a side discussion with an other teammate, it would be preferable to rename the file to openssl-lts (like we are doing with nodejs and nodejs-lts)

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### openssl-v1-lts

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### openssl-lts

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
